[DOCU-2807] Runtime Configuration API Externalization (#4898)
Kong/docs.konghq.com: src/gateway/admin-api/index.md

---

{% unless page.edition == "gateway" %}

> The API for configurating Kong Konnect Runtime Groups

Reviewer: ```suggestion
The API for configuring Kong Konnect Runtime Groups
```

## Servers

| URL                | Description                                                                                                                         |
| ---------                | -----------                                                                                                                         |
| `https://us.api.konghq.com/v2/runtime-groups/`                   | US Region Konnect Platform Base URL page.                                                                                                                 |
| `https://eu.api.konghq.com/v2/runtime-groups/{runtime_group_id}` | EU Region Konnect Platform Base URL |
| `https://us.kong-admin.konghq.com/v1/{runtime_group_id}` | US Region Kong Admin Base URL |
|`https://eu.kong-admin.konghq.com/v1/{runtime_group_id}` | EU Region Kong Admin Base URL|
       

> You can generate a Personal Access Token (PAT) from the personal access token page in the Konnect dashboard. The PAT token must be passed in the header of a request, for example: `curl -X GET https://global.api.konghq.tech/v2/users/' --header 'Authorization: Bearer kpat_xgfT`

Reviewer: ```suggestion
You can generate a Personal Access Token (PAT) from the personal access token page in the {{site.konnect_short_name}} dashboard. The PAT token must be passed in the header of a request, for example: `curl -X GET https://global.api.konghq.tech/v2/users/' --header 'Authorization: Bearer kpat_xgfT`
```

Provide your bearer token in the Authorization header when making requests to protected resources.

Example: `Authorization: Bearer 123`

This API is similar to the [Kong Gateway admin API](/gateway/admin-api/) with a few major differences:

* No `PATCH` methods.
* Non-backwards compatible error responses.
* `POST` methods do not support URL encoded forms.

--- 
## Nodes
### List Runtime Instance Records

> Returns a list of runtime instance records that are associated to this runtime group. A runtime instance record contains all the metadata information of the Kong Gateway dataplane.

Reviewer: ```suggestion
Returns a list of runtime instance records that are associated with this runtime group. A runtime instance record contains all the metadata information of the {{site.base_gateway}} dataplane.
```

**Endpoint**

<div class="endpoint get">/v2/runtime-groups/{runtime_group_id}/nodes</div>

**Path Parameters**

| Attribute                | Description                                                                                                                         |
| ---------                | -----------                                                                                                                         |
| `page.size`                   | The number of items to include in a page.                                                                                                                 |
| `page.number` | The specific page number in the collection results. |
       

**Response**

```
HTTP 200 OK
```

### Fetch Runtime Instance Record

> Retrieve a specific runtime instance record associated to this runtime group. A runtime instance record contains all the metadata information of the Kong Gateway dataplane.

Reviewer: ```suggestion
Retrieve a specific runtime instance record associated to this runtime group. A runtime instance record contains all the metadata information of the {{site.base_gateway}} dataplane.
```
